main/src/cgeo/geocaching/connector/oc/OkapiClient.java

```diff
@@ -435,7 +435,7 @@ private static void parseCoreCache(final ObjectNode response, final Geocache cac
         cache.setDifficulty((float) response.get(CACHE_DIFFICULTY).asDouble());
         cache.setTerrain((float) response.get(CACHE_TERRAIN).asDouble());
 
-        cache.setInventoryItems(response.get(CACHE_TRACKABLES_COUNT).asInt());
+        cache.mergeInventory(parseTrackables((ArrayNode) response.path(CACHE_TRACKABLES)));
```

Comment:
You're right. Yet, I remember I've checked that... I've missed something ;)
Moreover, Unit Tests were ok, some tests should be missing. I'll check to add them.